.travis.yml

```diff
@@ -8,9 +8,6 @@ cache:
         - $HOME/.composer/cache
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
```

Comment:
This does more than your ticket claims to do (you actually dropped PHP 5.5 support